runtime/queries/mojo/highlights.scm

```diff
@@ -0,0 +1,152 @@
+(attribute attribute: (identifier) @property)
```

Comment:
The highlights need to be updated to use Helix-specific capture names: https://docs.helix-editor.com/master/themes.html#syntax-highlighting

For example `@property` should become `@variable.other.member`, `(true)` and `(false)` should be captured as `@constant.builtin.boolean`, escape sequences as `@constant.character.escape`, etc.

Reply:
I've updated queries so that Mojo is a superset of the current Python queries.

Reply:
If it's a superset you can use a comment in the query files `; inherits: python` which will do the same thing as copy/pasting in the python query for that file

Reply:
If the python queries start using a new node that isn't in the mojo grammar then we will need to replace the `; inherits: python`. But if you just want to add onto the python queries you can place patterns before or after the inherits comment. The ones before are higher precedence and the ones after lower

Reply:
At the moment, the differences are mainly related to modifying queries rather than adding new ones. So, I think I'm done with the changes, further improvements depend on the tree-sitter repository.